google/cloud/bigquery/v2/minimal/internal/job_connection_test.cc

```diff
@@ -120,12 +186,28 @@ TEST(JobConnectionTest, GetJobTooManyTransients) {
       .WillRepeatedly(
           Return(Status(StatusCode::kDeadlineExceeded, "try-again")));
   auto conn = CreateTestingConnection(std::move(mock));
+
   GetJobRequest request;
   auto result = conn->GetJob(request);
   EXPECT_THAT(result,
               StatusIs(StatusCode::kDeadlineExceeded, HasSubstr("try-again")));
 }
 
+TEST(JobConnectionTest, ListJobsTooManyTransients) {
+  auto mock = std::make_shared<MockBigQueryJobRestStub>();
+  EXPECT_CALL(*mock, ListJobs)
+      .Times(AtLeast(2))
+      .WillRepeatedly(
+          Return(Status(StatusCode::kDeadlineExceeded, "try-again")));
+  auto conn = CreateTestingConnection(std::move(mock));
+
+  ListJobsRequest request;
+  auto range = conn->ListJobs(request);
+  auto begin = range.begin();
+  ASSERT_NE(begin, range.end());
+  EXPECT_EQ(StatusCode::kDeadlineExceeded, begin->status().code());
```

Comment:
nit: `EXPECT_THAT(*begin, StatusIs(StatusCode::kDeadlineExceeded));`

Reply:
oh, and there is another one on line 178. I forgot to add my comment to the review.